packages/apm/src/integrations/tracing.ts

```diff
@@ -853,7 +853,7 @@ export class Tracing implements Integration {
         Tracing._log(`[Tracing] Transaction: ${SpanStatus.Cancelled} since it maxed out maxTransactionDuration`);
         if (event.contexts && event.contexts.trace) {
           event.contexts.trace = {
-            ...event.contexts.trace,
+            ...(typeof event.contexts.trace === 'object' && event.contexts.trace),
```

Comment:
Why do we now do a type check here?

Reply:
Because `unknown` is more restrictive than `any`, and requires type check before it can be spread. On the other hand, it doesnt require unnecessary `no-explcit-any` linting rule.

Reply:
`event.contexts.trace` has type `Context`, it must be an object, mustn't it?

Reply:
Fixed